lib/pure/asyncdispatch.nim

```diff
@@ -572,7 +572,7 @@ when defined(windows) or defined(nimdoc):
     ## Sends `size` bytes from `buf` to `socket`. The returned future
     ## will complete once all data has been sent.
     ##
-    ## **WARNING**: Use it with caution. If `buf` refers to GC'ed object,
+    ## .. warning:: Use it with caution. If `buf` refers to GC'ed object,
```

Comment:
```suggestion
    ## .. warning:: Use it with caution. If `buf` refers to a memory managed object,
```

Because there was an idea is to move from "Garbage Collector" to "memory management" in documentation.

Reply:
"memory managed" doesn't sound right IMO